providers/nexmo/src/lib/nexmo.provider.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  ChannelTypeEnum,
+  ISendMessageSuccessResponse,
+  ISmsOptions,
+  ISmsProvider,
+} from '@notifire/core';
+
+import Vonage from '@vonage/server-sdk';
+
+export class NexmoSmsProvider implements ISmsProvider {
+  id = 'nexmo';
+  channelType = ChannelTypeEnum.SMS as ChannelTypeEnum.SMS;
+  private vonageClient: Vonage;
+
+  constructor(
+    private config: {
+      apiKey: string;
+      apiSecret: string;
+      from: string;
+    }
+  ) {
+    this.vonageClient = new Vonage({apiKey: config.apiKey, apiSecret: config.apiSecret});
```

Comment:
Could you please run prettier in this file? 💅 @matthiez is taking a look at why it's not working on the precommit hook